notebooks/7_reshape_table_layout.ipynb

```diff
@@ -0,0 +1,1654 @@
+{
```

Comment:
<p>Stating it this way makes it sound like the <code>no2</code> dataset already exists. I would rephrase it to indicate that you are filtering the <code>air_quality</code> DataFrame in order to create a new DataFrame which only contains the NO2 measurements.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>I'll move the filtering to the subsection where the data is effectively used instead of making it part of the introduction. </p>